frontend/src/metabase/components/DateExcludeWidget/DateExcludeWidget.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import { space } from "metabase/styled-components/theme";
+
+export const Container = styled.div`
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: space-between;
+  align-items: center;
+  border-top: 1px solid ${color("border")};
+  border-bottom: 1px solid ${color("border")};
+  padding: ${space(2)};
+`;
```

Comment:
Copy-pasted from [DatePickerFooter.styled.tsx](https://github.com/metabase/metabase/blob/519e2d327949de1de6b028bbaa22833806bf94c0/frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePickerFooter.styled.tsx#L7-L14), but with a `border-bottom`, and the same `padding` on all sides:

<img width="839" alt="Screenshot 2024-01-17 at 11 02 42 AM" src="https://github.com/kienstra/metabase/assets/4063887/bb13cd52-1797-45d9-9d15-53e230652fd2">